### PR TITLE
Feat/kadena cli/command tests

### PR DESCRIPTION
**Pull request by barthuijgen**

### CODE REVIEW

File: packages/tools/kadena-cli/src/tx/tests/createTransaction.test copy.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { describe, expect, it } from 'vitest';
+import { mockPrompts, runCommand } from '../../utils/test.util.js';
```
Comment (nillo) [inline packages/tools/kadena-cli/src/tx/tests/createTransaction.test copy.ts:2]: fix filename? copy.ts?

File: packages/tools/kadena-cli/src/utils/logger.ts [hunk 2]
```diff
@@ -77,7 +77,7 @@ export const defaultTransport: Transport = (record) => {
     [LEVELS.info]: stdErrChalk.white,
```
Comment (nillo) [inline packages/tools/kadena-cli/src/utils/logger.ts:77]: still doubting these color levels, and maybe use / create something for the inversion? when using black / white terminal. 

Maybe just black on white bg, and white on black bg

**Review by nillo — APPROVED** [review]
lgtm

File: packages/tools/kadena-cli/src/tx/tests/createTransaction.test copy.ts (hunk 1 above)
Comment (barthuijgen) [reply packages/tools/kadena-cli/src/tx/tests/createTransaction.test copy.ts:2]: Oh huh, never even noticed that 🤣 thanks

**Review by nillo — APPROVED** [review]

**Pull request merged**